Apps/Log/Log.xcodeproj/project.pbxproj

```diff
@@ -85,12 +85,20 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		E209BBBD281EF4D300C9FB93 /* DevResources */ = {
```

Comment:
Nem tudom hova irjam, ez az athelyezes nem igenyel valahol modositast a referenciakban?
[Log/.swiftlint.yml → Apps/Log/.swiftlint.yml](https://github.com/furry-fiesta/furry-fiesta-ios/pull/13/commits/fb0501789387cdcb0d8472587eca6e937cfd9d29#diff-0a652159f9e257b693294f2371e53b69bcea34b1f532babc43dce717f34d7f53)

Reply:
Nice catch, I just updated the  file inside of `Apps/Log/.swiftlint.yml` which has a reference to the root swiftlint file.